damus/Util/Hashtags.swift

```diff
@@ -9,62 +9,74 @@ import Foundation
 import SwiftUI
 
 struct CustomHashtag {
-    let name: String
+    let image_name: String
+    let display: String
     let offset: CGFloat?
     let color: Color?
     
-    init(name: String, color: Color? = nil, offset: CGFloat? = nil) {
-        self.name = name
+    init(image_name: String, display: String, color: Color? = nil, offset: CGFloat? = nil) {
+        self.image_name = image_name
+        self.display = display
         self.color = color
         self.offset = offset
     }
-    
-    static let coffee = CustomHashtag(name: "coffee", color: DamusColors.brown, offset: -1.0)
-    static let bitcoin = CustomHashtag(name: "bitcoin", color: Color.orange, offset: -3.0)
-    static let nostr = CustomHashtag(name: "nostr", color: DamusColors.purple, offset: -2.0)
-    static let plebchain = CustomHashtag(name: "plebchain", color: DamusColors.deepPurple, offset: -3.0)
-    static let zap = CustomHashtag(name: "zap", color: DamusColors.yellow, offset: -4.0)
 }
 
-
 let custom_hashtags: [String: CustomHashtag] = [
-    "bitcoin": CustomHashtag.bitcoin,
-    "btc": CustomHashtag.bitcoin,
-    "nostr": CustomHashtag.nostr,
-    "coffee": CustomHashtag.coffee,
-    "coffeechain": CustomHashtag.coffee,
-    "plebchain": CustomHashtag.plebchain,
-    "zap": CustomHashtag.zap,
-    "zaps": CustomHashtag.zap,
-    "zapathon": CustomHashtag.zap,
-    "onlyzaps": CustomHashtag.zap,
```

Comment:
can we just keep this style, change `name` to display name and lowercase it when we need to.

Reply:
nm! ignore this comment lol

Reply:
This doesn't have the CamelCase text in it though.